it/mail/chrome/messenger/addressbook/ldapAutoCompErrs.properties

```diff
@@ -91,48 +91,48 @@ errorAlertFormat=Codice di errore %1$S: %2$S\n\n %3$S
 
 ## @name TIMELIMIT_EXCEEDED_HINT
 ## @loc none
-10003=Riprovare o contattare l'amministratore del sistema.
+10003=Riprova più tardi o contattare il tuo amministratore del sistema.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

 +10003=Riprova più tardi o contatta l'amministratore del sistema.